dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraph.java

```diff
@@ -115,16 +119,32 @@ void addPath(DependencyPath path) {
     return result;
   }
 
-  /**
-   * @return a mutable copy of the paths in this graph, usually in breadth first order
-   */
+  /** Returns a mutable copy of the paths in this graph, usually in breadth first order. */
   public List<DependencyPath> list() {
     return new ArrayList<>(graph);
   }
 
   /**
-   * @return all paths to the specified artifact
+   * Returns dependency path of the root node.
+   *
+   * @throws IllegalStateException if the graph is empty
+   */
+  public DependencyPath getRootPath() {
+    if (graph.isEmpty()) {
+      throw new IllegalStateException("The graph is empty");
+    }
+    return graph.get(0);
+  }
+
+  /**
+   * Returns dependency paths from the root to the dependencies of the node at {@code
+   * dependencyPath}.
    */
+  public List<DependencyPath> getChildPaths(DependencyPath dependencyPath) {
```

Comment:
possibly rename dependencyPath --> node or perhaps dependencyPath --> parent

and now that I think about it, getChildren might be a better method name than getChildPaths

Reply:
Updated to getChildren. Thank you for suggestion.